oci/python/documentation.yaml

```diff
@@ -0,0 +1,23 @@
+version: 1
+
+application: Python
+is_chiselled: true
+description: |
+  Python is an interpreted, interactive, object-oriented, open-source
+  programming language.
```

Comment:
This plagiarizes the upstream's description. Let's change it

Reply:
Updated in https://github.com/canonical/oci-factory/pull/125/commits/41549c18435d3e992b8c779fe9a2071b25c0b415.